web/src/config-ui/subNavConfig.js

```diff
@@ -43,9 +43,10 @@ export default [
     tabName: "license",
     displayName: "License",
     to: (slug) => `/app/${slug}/license`,
-    displayRule: ({ app }) =>
+    displayRule: ({ app, isHelmManaged }) =>
```

Comment:
`subNavConfig` uses `displayRule` callback to determine if a tab should show. This already has `isHelmManaged` available to it so we just need to destructure the boolean and add it to the expression.